packages/discord.js/src/managers/CategoryChannelChildManager.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const CachedManager = require('./CachedManager');
+const GuildChannel = require('../structures/GuildChannel');
+
+/**
+ * Manages API methods for CategoryChannels' children and stores their cache.
+ * @extends {CachedManager}
+ */
+class CategoryChannelChildManager extends CachedManager {
+  constructor(channel, iterable) {
+    super(channel.client, GuildChannel, iterable);
+    /**
+     * The category channel this manager belongs to
+     * @type {CategoryChannel}
+     */
+    this.channel = channel;
+
+    /**
+     * The guild this manager belongs to
+     * @type {Guild}
+     */
+    this.guild = channel.guild;
+  }
+
+  /**
+   * Options for creating a channel using {@link CategoryChannel#createChannel}.
+   * @typedef {Object} CategoryCreateChannelOptions
+   * @property {ChannelType} [type=ChannelType.GuildText] The type of the new channel.
+   * @property {string} [topic] The topic for the new channel
+   * @property {boolean} [nsfw] Whether the new channel is NSFW
+   * @property {number} [bitrate] Bitrate of the new channel in bits (only voice)
+   * @property {number} [userLimit] Maximum amount of users allowed in the new channel (only voice)
+   * @property {OverwriteResolvable[]|Collection<Snowflake, OverwriteResolvable>} [permissionOverwrites]
+   * Permission overwrites of the new channel
+   * @property {number} [position] Position of the new channel
+   * @property {number} [rateLimitPerUser] The rate limit per user (slowmode) for the new channel in seconds
+   * @property {string} [rtcRegion] The specific region of the new channel.
+   * @property {string} [reason] Reason for creating the new channel
+   */
+
+  /**
+   * Creates a new channel within this category.
+   * <info>You cannot create a channel of type `GUILD_CATEGORY` inside a CategoryChannel.</info>
+   * @param {string} name The name of the new channel
+   * @param {CategoryCreateChannelOptions} options Options for creating the new channel
+   * @returns {Promise<GuildChannel>}
+   */
+  create(name, options) {
+    return this.guild.channels.create(name, {
+      ...options,
+      parent: this.channel.id,
+    });
+  }
+
```

Comment:
```suggestion

  /**
     * Batch-updates the category's channels' positions.
     * @param {ChannelPosition[]} channelPositions Channel positions to update
     * @returns {Promise<Guild>}
     * @example
     * channel.children.setPositions([{ channel: channelId, position: newChannelIndex }])
     *   .then(guild => console.log(`Updated channel positions for ${guild}`))
     *   .catch(console.error);
     */
  async setPositions(channelPositions) {
    channelPositions = channelPositions.map(r => ({
      id: this.client.channels.resolveId(r.channel),
      position: r.position,
      lock_permissions: r.lockPermissions,
      parent_id: typeof r.parent === 'undefined' ? this.channel.id : this.guild.channels.resolveId(r.parent),
    }));

    return this.guild.channels.setPositions(channelPositions);
  }

```

Reply:
What would be the benefit of having this here?

Reply:
> this.channel.id

This will set parent of a channel to the category channel when you will set positions.

Reply:
I mean I can see the use case but shouldn't this only work for the channels within the category? There's no way to prevent that from happening

Reply:
> I mean I can see the use case but shouldn't this only work for the channels within the category? There's no way to prevent that from happening

Do you mean it shouldn't move a channel from other categories?

Reply:
Yes. I'll wait for @discordjs/the-big-4 and commit if they think it's worth it